lib/internal/pushes.js

```diff
@@ -45,75 +46,71 @@ PushBullet.prototype.link = function link(deviceParams, title, url, callback) {
  * @param  {String}   filePath     Path to file.
  * @param  {String}   message      Optional message to send with file.
  * @param  {Function} callback     Callback for when the request is complete.
+ * @returns {Promise}
  */
-PushBullet.prototype.file = function file(deviceParams, filePath, message, callback) {
```

Comment:
A message/body is a valid parameter for file and shouldn't be removed. https://docs.pushbullet.com/#parameters-for-different-types-of-pushes